modules/statistics/php/stats_demographic.class.inc

```diff
@@ -163,65 +163,73 @@ class Stats_Demographic extends \NDB_Form
         foreach (\Utility::getProjectList() as $key => $value) {
             $projects[$key] = $value;
         }
-        $currentProject = null;
-        if ($_REQUEST['DemographicProject'] ?? ''
-        ) {
-            $currentProject = htmlspecialchars($_REQUEST['DemographicProject']);
+
+        $demographicProject = $_REQUEST['DemographicProject'] ?? null;
```

Comment:
Here you make the variable `null` when not set and elsewhere you set it to the empty string. I think it would be better to choose just one unless the difference is significant. If that's the case, a comment explaining why would be good.